app/dashboard/template.hbs

```diff
@@ -0,0 +1,238 @@
+                            <div class="col-md-3 text-center m-v-sm">
+                                <div>
+                                    <a role="button" href="/prereg/" class="btn btn-success btn-success-high-contrast f-w-xl">{{t 'banners.prereg.button'}}</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="quickSearch">
+        <div class="container p-t-lg">
+            <div class="row m-t-lg">
+                <div class="col-md-10 col-md-offset-1 col-lg-8 col-lg-offset-2">
+                    <div class="row">
+                        <div class="col-xs-12">
+                            <h2 class='col-xs-9'>{{t 'dashboard.title'}}</h2>
+                            <div class='m-b-lg col-xs-3'>
+                                <div class='pull-right'>
+                                    <button onclick={{action 'toggleModal'}} class='btn btn-success btn-success-high-contrast m-t-md f-w-xl'>{{t 'dashboard.create_new_project_button'}}</button>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-xs-12">
+                            <div class='row quick-project'>
+                                {{#if initialLoading}}
+                                    {{loading-indicator}}
+                                {{else}}
+                                    {{#if totalNodes}}
+                                        <div class='col-xs-12'>
+                                            <div class='m-b-sm text-center'>
+                                                <div class="m-v-sm quick-search-input">
+                                                    <input class='form-control' placeholder="{{t 'dashboard.quicksearch.search'}}" type="text" oninput={{perform filterNodes value="target.value"}}>
+                                                </div>
+                                            </div>
+                                            <p class="text-center f-w-lg">{{t 'dashboard.quicksearch.other_links'}}</p>
+                                            <div class='quick-search-table'>
+                                                <div class="row node-col-headers m-t-md">
+                                                    <div class="col-sm-3 col-md-6">
+                                                        <div class="quick-search-col">
+                                                            {{t 'general.title'}}
+                                                            {{sort-button sortAction=(action 'sort') curOrder=sortOrder curBy=sortBy sortOrder='asc' sortBy='title'}}
+                                                            {{sort-button sortAction=(action 'sort') curOrder=sortOrder curBy=sortBy sortOrder='desc' sortBy='title'}}
```

Comment:
Minor: Since these always come in asc/desc pairs, seems like it could be one `{{sort-buttons}}` for both